examples/windows/GetUpTimeFromSystemLogPastDay.ps1

```diff
@@ -0,0 +1,37 @@
+[timespan]$downTime = New-TimeSpan -start 0 -end 0
+[timespan]$totalDownTime = New-TimeSpan -start 0 -end 0
+Get-EventLog -LogName system | 
+Where-Object `
+{ $_.eventid -eq 6005 -OR $_.eventID -eq 6006 -AND $_.timegenerated -gt (get-date).adddays(-1) } | 
+Sort-Object -Property timegenerated |
+Foreach-Object `
+{
+  if ($_.EventID -eq 6006)
+     { 
+       $down = $_.TimeGenerated
+     } #end if eventID
+  Else
+     { 
+      $up = $_.TimeGenerated 
+     } #end else
+   if($down -AND $up)
+     {
+      if($down -ge $up) 
+         { 
+           Write-Host -foregroundColor Red "*** Invalid data. Ignoring $($up)"
```

Comment:
@Kav91 - this is going to cause a broken event right? writing to STDOUT without being in JSON